src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -122,9 +124,29 @@ def _update_request_data_increased_size_limit(http_info: dict, max_size: int) ->
     )
 
 
+def get_lumigo_connection_id(instance) -> Optional[int]:
+    return getattr(instance, LUMIGO_CONNECTION_ID_KEY, None)
+
+
+def set_lumigo_connection_id(instance):
+    r = random.random()
+    print("Choose random", r)
+    setattr(instance, LUMIGO_CONNECTION_ID_KEY, r)
+
+
 #   Wrappers  #
 
 
+def _http_init_wrapper(func, instance, args, kwargs):
```

Comment:
Won't we have an issue with sync calls that use the same `HTTPConnection` object?